### PR TITLE
Expose the same API for `define.amd`.

### PR DESCRIPTION
**Pull request by rwjblue**

### CODE REVIEW

**Comment by kratiahuja** [comment]
LGTM

File: addon/index.js [hunk 1]
```diff
@@ -27,6 +27,9 @@ export {
 if (typeof define !== 'undefined' && define.amd){
   define('ember-inflector', ['exports'], function(__exports__){
     __exports__['default'] = Inflector;
+    __exports__.pluralize = pluralize;
+    __exports__.singularize = singularize;
+
     return Inflector;
   });
 } else if (typeof module !== 'undefined' && module['exports']){
```
Comment (kratiahuja) [inline addon/index.js]: We also need to expose singularize pluralize here as well right?

Comment (kratiahuja) [inline addon/index.js]: When this check is true, we also need to export singularize/pluralize.

Probably better to do:
```javascript
module['exports'] = {
  'Inflector': Inflector,
  'singularize': singularize,
  'pluralize': pluralize
}
```

thoughts?

**Comment by rwjblue** [comment]
Ya, this is basically the CJS case. I wonder if we have any actual CJS users? Either way, it would be a breaking change so we should decouple somewhat if we have to change to a POJO instead of a single main.

**Comment by kratiahuja** [comment]
I don't think we have CJS users but would be nice to add a todo here 😄

File: addon/index.js (hunk 1 above)
Comment (rwjblue) [inline addon/index.js]: Updated to expose `singularize` and `pluralize` support in CJS mode...

**Comment by bmac** [comment]
Thanks for working on this @rwjblue.

**Comment by kratiahuja** [comment]
LGTM.

**Comment by msranade** [comment]
Thanks @rwjblue and @kratiahuja for fixing this.

**Pull request merged**